cmd/otelcol/Dockerfile

```
@@ -1,7 +1,8 @@
-FROM alpine:3.17.0 as certs
+ARG DOCKER_REPO
```

```suggestion
ARG DOCKER_REPO=docker.io
```
otherwise I think the default is invalid. Or may be better to require the registry to end w/ `/` so the default is from the system.